data_steward/cdr_cleaner/cleaning_rules/recent_concept_suppression.py

```diff
@@ -0,0 +1,251 @@
+"""
+Suppress Non-PPI Concepts created within past year
+
+Original Issues: DC-1692, DC-2789
+"""
+
+# Python imports
+import logging
+from datetime import datetime
+
+# Project imports
+from cdr_cleaner.cleaning_rules.deid.concept_suppression import AbstractBqLookupTableConceptSuppression
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+from common import JINJA_ENV, CDM_TABLES, DEFAULT_CONCEPT_VALID_START_DATE
+from utils.bq import validate_bq_date_string, list_tables
```

Comment:
This `list_tables` import is not needed because the definition that is actually getting used is:
`BigQueryClient.list_tables`.